content/docs/started/k8s/kfctl-istio-dex.md

```diff
@@ -1,21 +1,20 @@
 +++
-title = "Multi-user, auth-enabled Kubeflow with kfctl_existing_arrikto"
-description = "Instructions for installing Kubeflow with kfctl_existing_arrikto.yaml config"
+title = "Multi-user, auth-enabled Kubeflow with kfctl_istio_dex"
+description = "Instructions for installing Kubeflow with kfctl_istio_dex.yaml config"
 weight = 4
 +++
 
 Follow these instructions if you want to install Kubeflow on an existing Kubernetes cluster.
 
-This installation of Kubeflow is maintained by
-[Arrikto](https://www.arrikto.com/), it is geared towards existing Kubernetes
+This installation of Kubeflow is geared towards existing Kubernetes
 clusters and does not depend on any cloud-specific feature.
 
```

Comment:
Add a heading here (immediately after the new note suggested in my previous comment) to break up the long introduction:

```
## Overview
```

Reply:
Sure I'll include this header.

Reply:
The word overview alone isn't rendering properly on the website. I've chosen to go with Architecture Overview instead

Reply:
Ah yes, I've seen that before. Strange!